contents/tutorials/migrate-eu-cloud.md

```diff
@@ -0,0 +1,54 @@
+---
+title: How to migrate to EU Cloud
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-10-10
+featuredImage: ../images/tutorials/banners/migrate-eu-cloud.png
+topics: ['configuration']
+---
+
+> **Warning:** Migrating event data to PostHog's EU Cloud is a time consuming process, likely to take several days or weeks, and there is currently no way to migrate insights or dashboards. We strongly recommend that users with a large amount of event data begin by creating a new EU cloud instance and migrate data at a later point, if strictly needed. 
```

Comment:
It's probably worth explicitly stating the objects that aren't supported via migrator:

- user accounts
- actions
- cohorts
- insights
- dashboards
- annotations
- project settings (eg filters for internal users, api keys)
**- are group properties migrated?**

However these objects are accessible via API and you can decide whether automating this via script, or doing it manually, is warranted once you're up and running on EU-Cloud.


Otherwise, this looks good to me.

Reply:
Updated to add the list and stronger warnings throughout. @camerondeleone